app/screens/Home.js

```diff
@@ -0,0 +1,13 @@
+import React, { Component } from 'react'
```

Comment:
```suggestion
import React from 'react'
import {
  StyleSheet,
  View,
  Text
} from 'react-native'
import { colors } from 'app/config'

const Home = ({ navigation }) => {

  return (
    <View style={styles.container}>
      <Text style={styles.text}>
        Some Text
      </Text>
    </View>
  )
}

UserProfile.navigationOptions = {
  title: `Hand Wash Timer`
}

export default Home

const styles = StyleSheet.create({
  container: {
    flex: 1,
  },
  text: {
    color: colors.black,
    fontSize: 18,
    textTransform: 'capitalize',
    flex: 1,
  }
})
```

I'd rather see modern react conventions across the codebase - functional vs class components. I find for a placeholder Home screen something like this is useful and sets out a coding standard/approach? Do you agree or prefer your approach?